ecs_deploy/cli.py

```diff
@@ -23,6 +23,56 @@ def get_client(access_key_id, secret_access_key, region, profile):
     return EcsClient(access_key_id, secret_access_key, region, profile)
 
 
+@click.command()
+@click.argument('cluster')
+@click.argument('service')
+@click.option('-t', '--tag', help='Changes the tag for ALL container images')
+@click.option('-i', '--image', type=(str, str), multiple=True, help='Overwrites the image for a container: <container> <image>')
+@click.option('-c', '--command', type=(str, str), multiple=True, help='Overwrites the command in a container: <container> <command>')
+@click.option('-e', '--env', type=(str, str, str), multiple=True, help='Adds or changes an environment variable: <container> <name> <value>')
+@click.option('-s', '--secret', type=(str, str, str), multiple=True, help='Adds or changes a secret environment variable from the AWS Parameter Store (Not available for Fargate): <container> <name> <parameter name>')
+@click.option('-r', '--role', type=str, help='Sets the task\'s role ARN: <task role ARN>')
+@click.option('--task', type=str, help='Task definition to be deployed. Can be a task ARN or a task family with optional revision')
```

Comment:
I think the description is misleading. The task is _not_ deployed and only updated. Or did I get that wrong?